packages/animations/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [1.0.0+4] - February 21, 2020
+
+* Fix a typo in the changelog dates
+* Add a note to the README that the `animations` package is not currently
+  compatible with Flutter's `stable` branch.
+
```

Comment:
Include that the example app has been switched back to the old style material stuff to be compatible with stable for now.